x-pack/plugins/cases/public/components/case_form_fields/assignees.tsx

```diff
@@ -40,14 +40,14 @@ interface Props {
   isLoading: boolean;
 }
 
+type UserProfileComboBoxOption = EuiComboBoxOptionOption<string> & UserProfileWithAvatar;
+
 interface FieldProps {
-  field: FieldHook;
-  options: EuiComboBoxOptionOption[];
+  field: FieldHook<CaseAssignees>;
+  options: UserProfileComboBoxOption[];
   isLoading: boolean;
   isDisabled: boolean;
   currentUserProfile?: UserProfile;
-  selectedOptions: EuiComboBoxOptionOption[];
-  setSelectedOptions: React.Dispatch<React.SetStateAction<EuiComboBoxOptionOption[]>>;
```

Comment:
These are the basic changes in the component. I get the selected options from the `field.value`. This is needed to be able to set the assignees from templates.